en/02_Developer_Guides/03_Forms/How_Tos/04_Create_a_GridField_ActionProvider.md

```diff
@@ -69,16 +72,17 @@ class GridFieldCustomAction extends AbstractGridFieldComponent implements GridFi
     public function getColumnContent($gridField, $record, $columnName) 
     {
         if (!$record->canEdit()) {
-            return;
+            return null;
         }
 
         $field = GridField_FormAction::create(
             $gridField,
-            'CustomAction'.$record->ID,
-            'Do Action',
-            "docustomaction",
+            'CustomAction' . $record->ID,
+            'Custom Action',
+            'docustomaction',
             ['RecordID' => $record->ID]
         );
+        $field->addExtraClass('btn btn-outline-dark');
```

Comment:
could be confusing why this is here
```suggestion
```

Reply:
I'll add a comment to explain it. Not having the classes there gives you a hideous unstyled button, and I can see people writing new CSS for it, without thinking to use the existing styles.

Reply:
Done